cloudferrylib/os/network/neutron.py

```diff
@@ -334,7 +334,8 @@ def convert_networks(self, net, cloud):
                                              'name',
                                              'shared',
                                              'tenant_name',
-                                             'router:external')
+                                             'router:external',
+                                             'provider:segmentation_id')
```

Comment:
I am not sure it is enough. We can have 2 networks with the same segmentation IDs, but with different types (f.e. `gre` and `vxlan`). And with this implementation CF will not care about it and consider networks are equal, but IRL they will be different (and will not overlap with each other).